README.md

```diff
@@ -3,6 +3,30 @@
 
 ## Introduction
 
-A command line interface to interact with EdgeX microservices. Replaces the need to manually construct complex CURL commands and/or maintain developer scripts.
+EdgeX-CLI is a command line interface tool meant for developers to interact with EdgeX microservices. It replaces the need to manually construct complex cURL commands and/or maintain developer scripts.
```

Comment:
I suggest completely removing the second sentence or changing it to saw it acts as a wrapper for the REST APIs.

Reply:
Removed - and updated